presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestEliminateCrossJoins.java

```diff
@@ -203,6 +203,10 @@ private JoinNode join(PlanNode left, PlanNode right, String... symbols)
                 left,
                 right,
                 criteria.build(),
+                ImmutableList.<Symbol>builder()
```

Comment:
Yes, is should be part of `Add outputSymbols support to JoinNode and optimizers`